src/symm.cc

```diff
@@ -168,7 +173,7 @@ void symm(
                     Acol_0.releaseLocalWorkspace();
 
                     std::set<ij_tuple> tile_set;
-                    for (int64_t i = 0; i < A.mt(); ++i) {
+                    for (int64_t i = 1; i < A.mt(); ++i) {
```

Comment:
It looks like using `for (int64_t i = 0;` would not cause problems.
Perhaps that is the more conservative choice for releasing remote workspace?

Reply:
I literally copied code from hemm. I don't know why it was set to 1 in hemm, but presumably Kadir had some reason. He changed it to 1 in 99b6491. I might say it's more conservative to _not_ release a tile, if it might be needed later. Everything should be cleaned up at the end.